prime-router/metadata/schemas/FL/fl-covid-19.schema

```diff
@@ -347,23 +359,20 @@ elements:
     # column BH
     - name: pregnant
       csvFields: [ { name: Pregnancy Status } ]
+      altValues:
+          - display: Pregnant
+            code: Y
+          - display: Not Pregnant
+            code: N
+          - display: Unknown
+            code: U
 
     # column BI
     # nonstandard
     - name: student_teacher_other
```

Comment:
It would be _really_ nice to just add Student and Teacher to patient_role - I think this would be easier for SimpleReport too - they don't want to have two different dropdown, one for student/teacher and one for staff/resident, ?